rome/src/test/resources/atom_1.0.xml

```diff
@@ -17,6 +17,7 @@
     <uri>http://example.com</uri>
     <email>author1@example.com</email>
   </contributor>
+  <icon>http://example.com/icon</icon>
```

Comment:
It's implicitly tested by `TestOpsAtom10.testWireFeedSyndFeedConversion` which does a round-trip from the entire `SyndFeed` to `WireFeed` and back.